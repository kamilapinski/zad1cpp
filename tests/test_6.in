Poland 1
Germany 1
America 1
Brazil 1
China 2
Peru 3
Maroco 2
France 3
Argentina 2
Algeria 2
Germany 1
Mexico 2
Canada 3
=1 1 1
=3 2 1
=1 2 3
=1 2 1
=2 1 1
=1 1 2
=999 1 1
=1 1 999
=1 999 1