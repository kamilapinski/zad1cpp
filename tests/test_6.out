1. Germany
2. Algeria
2. America
2. Argentina
2. Brazil
2. Canada
2. China
2. France
2. Maroco
2. Mexico
2. Peru
2. Poland
1. Germany
2. America
2. Brazil
2. Poland
5. Algeria
5. Argentina
5. China
5. Maroco
5. Mexico
10. Canada
10. France
10. Peru
1. Canada
1. France
1. Peru
4. Algeria
4. Argentina
4. China
4. Germany
4. Maroco
4. Mexico
10. America
10. Brazil
10. Poland
1. Algeria
1. Argentina
1. China
1. Germany
1. Maroco
1. Mexico
7. America
7. Brazil
7. Canada
7. France
7. Peru
7. Poland
1. Germany
2. America
2. Brazil
2. Poland
5. Algeria
5. Argentina
5. Canada
5. China
5. France
5. Maroco
5. Mexico
5. Peru
1. Canada
1. France
1. Germany
1. Peru
5. Algeria
5. America
5. Argentina
5. Brazil
5. China
5. Maroco
5. Mexico
5. Poland
1. Germany
2. America
2. Brazil
2. Poland
5. Algeria
5. Argentina
5. Canada
5. China
5. France
5. Maroco
5. Mexico
5. Peru
1. Canada
1. France
1. Peru
4. Germany
5. Algeria
5. America
5. Argentina
5. Brazil
5. China
5. Maroco
5. Mexico
5. Poland
1. Algeria
1. Argentina
1. China
1. Maroco
1. Mexico
6. Germany
7. America
7. Brazil
7. Canada
7. France
7. Peru
7. Poland
